### PR TITLE
development Branch

### PR DESCRIPTION
**Pull request by Kidd254**
The changes introduced to this branch include addition of the html and css files, addition linter configuration files, rectifying errors as identified by linter, addition of descriptive README files.

### CODE REVIEW

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @Kidd254 , 

Great work on making the changes requested by a previous reviewer 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights 💪 

- No linter errors
- correct GitHub flow is used

 ## Required Changes ♻️

- [ ] It is not correct to open a new pull request because you accidentally closed it because there is a way you can reopen a pull request. Please read [this gist](https://gist.github.com/Farhan-Haseeb/fd4cbb476de4e2931c322d7bacb75e5f) to learn more about it. However, as you have already worked on some tasks, all you can do is reference the closed pull request inside this Pull request summary.
- [ ] Your Pull request summary is not descriptive. Please add some description. Normally, this is where you list the changes introduced in the branch so that reviewers can know what to check very easily 😃 👍 
_Check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL] Please remove all unnecessary files like the javascript file which don't play any role in the project because all they add is complexity 😃 🏃 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,31 +1,35 @@
...
 
+#Getting Started
+To get a local copy up and running follow these simple example steps.
+Prerequisites
+Setup
+Install
+Usage
+Run tests
+Deployment
+Authors
```
Comment (Olamarx) [inline README.md:13]: - [ ] Just as the previous reviewers pointed, with patience copy the template [here](https://github.com/microverseinc/readme-template)  and paste it into your readme file. Then change the description, project name and all others to fit in for your project.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Kidd254, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

- [ ] Kindly add an organized description which has heading and the things you did in this very branch.

 ## Highlights


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,31 +1,38 @@
...
 
-👤 Author1
+👤@kidd254
 
-    GitHub: https://github.com/Kidd254
-    Twitter:https://twitter.com/lawrenc98789206
-    LinkedIn: https://www.linkedin.com/in/lawrence-kioko-972035240/
+    GitHub: @Kidd254
+    Twitter: @lawrence kioko
+    LinkedIn: Lawrence Kioko
```
Comment (christianbiring1) [inline README.md:22]: [OPTIONAL]
- [ ] Kindly make sure you add the social handler in the right way. 
`hint: Github: [Kidd254]( https://github.com/Kidd254)`

**Review by christianbiring1 — APPROVED** [review]
Hi @Kidd254 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
APPROVED

## Highlights

## Optional suggestions
- [ ] Linters are passing
- [ ] Descriptive project summary
- [ ] Implement all of the required changes by the previous reviewers

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Kidd254) [reply README.md:22]: @christianbiring1 thank you so much for the feedback

**Pull request merged**